biz.aQute.bndlib/src/aQute/bnd/exporter/runbundles/RunbundlesExporter.java

```diff
@@ -33,23 +36,35 @@ public RunbundlesExporter() {}
 	public Entry<String, Resource> export(String type, Project project, Map<String, String> options) throws Exception {
 		project.prepare();
 		Collection<Container> runbundles = project.getRunbundles();
-		Jar jar = new Jar(project.getName());
+		String prefix = options.getOrDefault(TARGET_DIR, "");
+		if (!prefix.endsWith("/")) {
+			prefix = prefix.concat("/");
+		}
+		String template = options.get(TEMPLATE);
+
+		Jar jar = null;
+		if (template == null) {
+			jar = new Jar(project.getName());
+		} else {
+			jar = new Jar(project.getName(), template);
```

Comment:
```suggestion
			jar = new Jar(project.getName(), project.getFile(template));
```

This will resolve a relative path against the base of the project.

Reply:
makes sense. done!